.circleci/config.yml

```diff
@@ -283,3 +289,22 @@ jobs:
           name: Submit coverage data to codecov.
           command: codecov
           when: always
+  style-check:
+    docker:
+      - image: python:3.6-alpine
+    steps:
+      - checkout
+      - run:
+          name: Install git
+          command: |
+            apk add git
+      - run:
+          name: Install autopep8
+          command: |
+            pip install autopep8
+      - run:
+          find gapic tests -name "*.py" | xargs autopep8 --in-place
```

Comment:
I see this is a valid shorthand syntax for a run step, but would it be helpful to also give it a name for consistency/descriptiveness.

Reply:
Done.